nixos/modules/security/acme.nix

```diff
@@ -732,7 +717,7 @@ in {
 
       certs = mkOption {
         default = { };
-        type = with types; attrsOf (submodule certOpts);
+        type = with types; attrsOf (submodule [ (inheritableModule false) certOpts ]);
```

Comment:
Thanks for the update @Infinisil ! I never thought of using `inherit (function call) resultValues` before.

I don't quite follow what is happening on this line though. Comparing the diff, am I correct in saying that the attrs in both inheritableModule + certOpts are being recursively merged? No objections to this, it's a much cleaner solution to what I was doing.